src/Pages/Login/LoginTap/AdminLogin/AdminLogin.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import './AdminLogin.scss';
+
+const AdminLogin = () => {
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+
+  const saveUserPW = e => {
+    setPw(e.target.value);
+  };
+
+  const goToMain = e => {
+    e.preventDefault();
+
+    fetch('https://f29c-211-106-114-186.jp.ngrok.io/user/signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        text: id,
+        password: pw,
+      }),
+    })
+      .then(response => {
+        if (response.ok === true) {
+          return response.json();
+        } else {
+          throw new Error('응답을 받을수 없음!');
+        }
+      })
+      .catch(error => {
+        // console.log(error);
+        alert('아이디 또는 비밀번호가 틀렸습니다.');
+      })
+      .then(data => {
+        // console.log(data);
+
+        if (data.message === 'login success') {
+          localStorage.setItem('TOKEN', data.token);
+          navigate('/');
+        }
+      });
+  };
+
+  return (
+    <div>
```

Comment:
```suggestion
    <div className="adminLogin">
```
scss를 사용할 때 최상위 부모태그에는 컴포넌트의 이름과 동일하게 작성해주세요